src/Workspaces/CoreTestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -24,10 +24,11 @@ internal sealed class InProcRemoteHostClient : RemoteHostClient, IRemoteHostServ
         private readonly HostWorkspaceServices _services;
```

Comment:
i'm curious that we have an InProcRemoteHostClient defined in teh test layer.  Don't we also have such a thing defined in teh real workspace layer (for when we're not using OOP).  I would have thought we could use the same guy for both.

Reply:
I don't think we want to use this kind of client for product. Using it inherently means calls are serialized. That's good for testing serialization of the remote calls but it's not good for product services running in-proc.